Challo/Challo/Common/Utilities/CustomDateFormatter.swift

```diff
@@ -50,4 +50,11 @@ struct CustomDateFormatter {
         return formatter.string(from: date)
     }
 
+    /// Formats a given date into the given template. Template follows the standard date formatting syntax (go Google).
+    static func displayDateWithCustomFormat(_ date: Date, template: String) -> String {
+        formatter.dateFormat = DateFormatter.dateFormat(fromTemplate: template,
+                                                        options: 0, locale: Locale(identifier: "en-SG"))
```

Comment:
umm i dont think u wan to limit to SG since this is gonna be used in india

Reply:
I did it that way because right now all the locales are set to SG, would you rather an additional argument for the locale identifier?

Reply:
umm i think for now its fine